Kentico.Kontent.Delivery/DeliveryClient.cs

```diff
@@ -308,15 +314,24 @@ private async Task<ApiResponse> GetResponseContentAsync(HttpResponseMessage http
                 return new ApiResponse(httpResponseMessage.Content, hasStaleContent, continuationToken, requestUri);
             }
 
-            string faultContent = null;
+            Error error;
 
             // The null-coalescing operator causes tests to fail for NREs, hence the "if" statement.
             if (httpResponseMessage?.Content != null)
             {
-                faultContent = await httpResponseMessage.Content.ReadAsStringAsync();
+                var faultContent = await httpResponseMessage.Content.ReadAsStringAsync();
+                error = JsonConvert.DeserializeObject<Error>(faultContent);
```

Comment:
use the instance of `JsonSerializer` (called Serializer) instead...

we try not to rely on the static classes

Reply:
Serializer used.